src/wasm-lib/kcl/src/std/math.rs

```diff
@@ -48,91 +48,19 @@ fn inner_rem(num: f64, divisor: f64) -> f64 {
 /// Compute the cosine of a number (in radians).
 pub async fn cos(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, KclError> {
     let num = args.get_number()?;
-    let result = inner_cos(num)?;
-
-    Ok(args.make_user_val_from_f64_with_type(result, NumericType::count()))
-}
-
-/// Compute the cosine of a number (in radians).
-///
-/// ```no_run
-/// exampleSketch = startSketchOn("XZ")
-///   |> startProfileAt([0, 0], %)
-///   |> angledLine({
-///     angle = 30,
-///     length = 3 / cos(toRadians(30)),
-///   }, %)
-///   |> yLineTo(0, %)
-///   |> close()
-///  
-/// example = extrude(exampleSketch, length = 5)
-/// ```
-#[stdlib {
-    name = "cos",
-    tags = ["math"],
-}]
-fn inner_cos(num: f64) -> Result<f64, KclError> {
-    Ok(num.cos())
+    Ok(args.make_user_val_from_f64_with_type(num.cos(), NumericType::count()))
```

Comment:
Shouldn't this be Radians not Count numeric type?

Reply:
I don't think so - this is the type of the result of cos, which is a ratio (if I remember high school maths properly :-)) so should be a Count